.gitignore

```diff
@@ -0,0 +1 @@
+.project
```

Comment:
@dystewart done.

Reply:
Note that you can create a local `.gitignore` file by putting things to ignore in `.git/info/exclude`.

Reply:
...and as long as I'm commenting here, rather than creating a new "Removed .gitignore" commit in this pull request, consider just updating the original commit and updating the PR.

Reply:
...and as long as I'm commenting here, rather than creating a new "Removed .gitignore" commit in this pull request, consider just updating the original commit and updating the PR.